contrib/Makefile.am

```diff
@@ -1,2 +1,2 @@
-SUBDIRS = bash_completion.d dracut initramfs pyzfs zcp
-DIST_SUBDIRS = bash_completion.d dracut initramfs pyzfs zcp
+SUBDIRS = bash_completion.d dracut $(ZFS_INITRAMFS) pyzfs zcp
+DIST_SUBDIRS = bash_completion.d dracut $(ZFS_INITRAMFS) pyzfs zcp
```

Comment:
Please add `initramfs` back to the `DIST_SUBDIRS` line.  This ensures that that `dracut` directory is always included in a `make dist` tarball even when it's not built as part of `make`.  See `etc/Makefile.am` for an example of this.